lib/src/model/hms_track.dart

```diff
@@ -44,7 +44,7 @@ class HMSTrack {
     List<HMSTrack> tracks = [];
 
     listOfMap.forEach((each) {
-      tracks.add(HMSTrack.fromMap(map: each));
+      tracks.add(HMSTrack.fromMap(map: each, isLocal: peer != null));
```

Comment:
why `peer != null` check is present?
Shouldn't it be `peer.isLocal` if the peer exists?